tcms/templates/include/tp_toolbar.html

```diff
@@ -0,0 +1,135 @@
+{% load i18n %}
+<style>
+    {#todo use css file#}
+    .kiwi-toolbar {
+        background: #fff;
+        padding-left: 45px;
+        padding-right: 15px;
+        display: flex;
+        justify-content: space-between;
+        align-items: center;
+    }
+
+    .kiwi-toolbar .find-and-reorder {
+        display: flex;
+        justify-content: space-between;
+        margin-right: 10px;
+    }
+
+    .find-and-reorder .btn-group {
+        padding-left: 10px;
+    }
+
+    .toolbar-separator {
+        border-right: 1px solid #d1d1d1;
+        padding-right: 10px;
+    }
+
+    .toolbar-search {
+        display: flex;
+        flex-direction: row-reverse;
+        flex-grow: 1;
+    }
+</style>
+<div class="kiwi-toolbar">
+    <div class="list-view-pf-checkbox">
+        <input type="checkbox" class="btn dropdown-toggle">
+    </div>
+    <div class="find-and-reorder toolbar-separator">
+        <div class="toolbar-pf-filter toolbar-separator">
+            <label class="sr-only" for="filter">Name</label>
+            <div class="input-group">
+                <div class="input-group-btn">
+                    <button type="button" class="btn btn-default dropdown-toggle" id="input-filter-button"
+                            data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">Name<span
+                            class="caret"></span></button>
+                    <ul class="dropdown-menu">
+                        <li class="selected"><a href="#">Name</a></li>
+                        <li><a href="#">Category</a></li>
+                        <li><a href="#">Priority</a></li>
+                        <li><a href="#">Author</a></li>
+                        <li><a href="#">Default Tester</a></li>
+                    </ul>
+                </div><!-- /btn-group -->
+                <input type="text" class="form-control" id="filter" placeholder="Filter By Name...">
+            </div><!-- /input-group -->
+        </div>
+        <div>
+            <div class="dropdown btn-group ">
+                <button type="button" class="btn btn-default dropdown-toggle" id="filter-button" data-toggle="dropdown"
+                        aria-haspopup="true" aria-expanded="false">Name<span class="caret"></span></button>
+                <ul class="dropdown-menu">
+                    <li class="selected"><a href="#">Name</a></li>
+                    <li><a href="#">Id</a></li>
+                    <li><a href="#">Status</a></li>
+                    <li><a href="#">Priority</a></li>
+                </ul>
+            </div>
+            <button class="btn btn-link" type="button">
+                <span class="fa fa-sort-alpha-asc"></span>
+            </button>
+        </div>
+    </div>
+    <div class="toolbar-separator">
```

Comment:
Order should be: New TestRun, Clone, Default tester, Status, Priority, Delete.

Delete could be separated with separators on both sides